### PR TITLE
WRN-6829: Add Camera Sample

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist
* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Add pattern-camera sample

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-6829

### Comments

### CODE REVIEW

**Review by seunghoh — CHANGES_REQUESTED** [review]
Besides the inline comment, let's 
1. change the repo name to pattern-ls2request-camera so that developer can understand this is luna service sample not, web media standard sample
2. Please add a comment at the top of the app.js file like "To use camera API, you need to XXXX for webOS permissions."

File: sandstone/pattern-camera/src/components/CameraView.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	let cameraOption;
+
+	useEffect(() => {
+		if (typeof window.PalmSystem !== 'undefined') {
+			dispatch(getCameraIds({}));
+		}
+	}, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+	useEffect(() => {
+		videoRef.current.load();
```
[the remark this answers is not in the document]
Comment (juwonjeong) [reply sandstone/pattern-camera/src/components/CameraView.js]: It's fixed

File: sandstone/pattern-ls2request-camera/src/components/CameraView.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+		if (typeof window === 'undefined' || typeof window.PalmSystem === 'undefined') {
+			return <div>This test will only function correctly on webOS systems!</div>;
+		}
+	};
+
+	if (typeof window.PalmSystem === 'undefined') {
+		return <div>This test will only function correctly on webOS systems!</div>;
+	}
+
+	cameraOption = escape(JSON.stringify({
```
Comment (seunghoh) [inline sandstone/pattern-ls2request-camera/src/components/CameraView.js:42]: I am not asking to replace now but please keep in mind that escape() will be deprecated. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape
FYI, :)

**Review by seunghoh — APPROVED** [review]
💯

**Pull request merged**